lib/menu.rb

```diff
@@ -0,0 +1,13 @@
+class Menu
+
+  attr_reader :list
+
+  def initialize
+    @list = {
```

Comment:
Each of the items in this list have similar attributes: a `name` and a `cost`. I wonder if we could extract these similarities out into a common class?